### ISSUE TITLE
How to use depth supervision method: depth-nerfacto?

### ISSUE THREAD

**Issue by robin32205** [nerfstudio-project/nerfstudio#1665]
hi, 
many thanks for the great work you done!
I had successfully trained a model based on my fisheye image dataset, using Nerfacto model.
But I want to use the depth supervision model (depth-nerfacto) to see the different. I already have the mesh model in obj file on the same coordinate system, how can I link to the training process? is there any introduction about this? :)

**Comment by matyja256** [comment]
Excuse me, have you made it now?

**Comment by rockywind** [comment]
How to train the depth-nerfacto when I have the lidar data?

**Comment by vijendra1125** [comment]
@rockywind Did you find anything on training depth-nerfacto with LiDAR data?

**Comment by vijendra1125** [comment]
In order to use depth-nerfacto, create depth image from lidar point. 
You could check [create_sfm_depth](https://github.com/nerfstudio-project/nerfstudio/blob/242c23f0f067064c16c49376c02271cd1cd2303b/nerfstudio/process_data/colmap_utils.py#L463) for reference. Also take a note on the issue [#2389](https://github.com/nerfstudio-project/nerfstudio/issues/2389).

**Comment by JiseungHong** [comment]
@vijendra1125 so, I am still getting black images when I run `ns-process-data images --use-sfm-depth`. What could be the solution?

**Comment by vijendra1125** [comment]
@JiseungHong Balck image should not be a problem. Image which you precieve as black image will not neccessarily have all pixel values as 0. Try to visualize the pixel values and you find that the depth calculated using SFM is saved in those images.

**Comment by JiseungHong** [comment]
@vijendra1125 Thanks for your response, even though the image turns out to be black, it seems that some pixels are not zero. My ultimate goal is to run `depth-nerfacto`. Will the `depth-nerfacto` be trained well if I add these depth-wise images into transforms.json as a value of the key `depth_file_path`?

**Comment by vijendra1125** [comment]
@JiseungHong, Yes it will. If you are using `ns-process-data` then `depth_file_path` should alread be added to the generated transforms.json

**Comment by JiseungHong** [comment]
@vijendra1125 Actually, I should've manually add the `depth_file_path` for the transforms.json file. Moreover, it didn't quite work well for training `depth-nerfacto`. It seems that there are some problems in the depth files... or maybe it might not be the best fit for training `depth-nerfacto`. Anyone has some ideas?

**Comment by vijendra1125** [comment]
@JiseungHong My bad, its been quite some time, seems like I didnt remember quite well if depth_file_path need added manually when ns-process-data is used. Regarding bad result, only guess I could make is that it could be simple because the depth image generated by SFM is very sparse. Other reason could be dataset itself depending on how well COLMAP works on it.
